db/dynamodb.py

```diff
@@ -296,3 +316,11 @@ def delete(self, Model: Type[T], k: str):
                 self.CONST.get_key(table_name): k
             }
         )
+
+    def delete_all(self, Model: Type[T]):
+        logging.info(f"Deleting {Model.__name__}")
+        table_name = self.CONST.get_table_name(Model)
+        table = self.ddb.Table(table_name)
+        table.delete()
+        table.wait_until_not_exists()
+        self.__create_table(table_name)
```

Comment:
May want to comment that this is the best way to remove all items from a table, because it makes fewer calls.